Tests/SentryTests/Integrations/Session/SentrySessionTrackerTests.swift

```diff
@@ -482,29 +615,28 @@ class SentrySessionTrackerTests: XCTestCase {
     private func assertInitSessionSent() {
         assertSessionInitSent(sessionStarted: fixture.currentDateProvider.date())
     }
-    
+
     private func assertSessionInitSent(sessionStarted: Date) {
-        if let session = fixture.client.captureSessionInvocations.last {
-            XCTAssertTrue(session.flagInit?.boolValue ?? false)
-            XCTAssertEqual(sessionStarted, session.started)
-            XCTAssertEqual(SentrySessionStatus.ok.description, session.status.description)
-            XCTAssertEqual(0, session.errors)
-            XCTAssertNil(session.timestamp)
-            XCTAssertNil(session.duration)
-            assertSessionFields(session: session)
-        } else {
-            XCTFail("No session init sent.")
+        guard let session = fixture.client.captureSessionInvocations.last else {
+            return XCTFail("No session init sent.")
         }
```

Comment:
`l`: We could also use XCTUnwrap, but then of course we need throws here and all the tests need throws. Up to you.